libcodechecker/server/client_db_access_server.py

```diff
@@ -189,10 +201,15 @@ def do_POST(self):
                 # Authentication requests must be routed to a different
                 # handler.
                 auth_handler = ThriftAuthHandler(self.manager,
-                                                 sess_token)
+                                                 auth_session, sess_token)
                 processor = codeCheckerAuthentication.Processor(auth_handler)
             else:
-                acc_handler = ThriftRequestHandler(session,
+                if auth_session:
+                    LOG.debug("Accessing as user " + auth_session.user)
+                else:
+                    LOG.debug("Unauthenticated access.")
+
+                acc_handler = ThriftRequestHandler(session, auth_session,
```

Comment:
If this variable list is so long already, I'd put `auth_session` into a new line.